src/main/java/dev/emi/trinkets/api/event/TrinketUnequipCallback.java

```diff
@@ -0,0 +1,25 @@
+package dev.emi.trinkets.api.event;
+
+import dev.emi.trinkets.api.SlotReference;
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.EventFactory;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.item.ItemStack;
+
+public interface TrinketUnequipCallback {
```

Comment:
Same notes here as equip callback, indent the lambda and document the link properly. additionally, "unequip" should be preferred over "un-equip"

Reply:
Same reply